cpp/include/raft/distance/detail/pairwise_matrix/kernel_sm60.cuh

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
Why the suffix `_sm60`?

Reply:
I am planning to add a version for Hopper (ie SM90). This mimics the code organization in CUTLASS. Since SM60 is the minimum supported version of RAFT, I named the kernel SM60.